src/test/java/com/trycloud/stepDefinitions/Dashboard_stepDef_TD.java

```diff
@@ -28,7 +32,7 @@ public void user_should_be_able_to_see_following_modules(List<String> expectedMo
             //then assert isDisplayed or not
             assertTrue(dash.modules(expectedModule).isDisplayed());
         }
-        //i could not extract text, why?
+        //I could not extract text, why?
```

Comment:
module element is the "a" tag and it does not have the text name of the module, if you want to extract text, you should target the child span tag under the "a" tag

Reply:
It doesn't work, i tried many ways from many parents , but to no veil